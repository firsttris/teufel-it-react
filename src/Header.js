import React, { Component } from 'react';
import Paper from './assets/images/backgrounds/paper.png';

class Header extends Component {
  render() {
    return (
      <div style={{position: 'absolute', top: '200px', width: '100%',}}>
        <div style={{backgroundImage: `url(${Paper})`, padding: '20px', display: 'inline-block' }}>
          <div style={{fontSize: '35px', letterSpacing:'10px',color:'#f00'}}>{this.props.title}</div>
          <div className="pt-3" style={{fontSize: '15px', letterSpacing:'5px',color:'white'}}>{this.props.subtitle}</div>
        </div>
      </div>
    );
  }
}

export default Header;
