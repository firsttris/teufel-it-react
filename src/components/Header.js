import React, { Component } from 'react';

class Header extends Component {
  render() {
    return (
      <div style={{position: 'absolute', top: '200px', width: '100%',}}>
        <div className="p-3" style={{backgroundColor: '#4B4B4B', display: 'inline-block' }}>
          <div style={{fontSize: '35px', letterSpacing:'8px',color:'#f00'}}>{this.props.title}</div>
          <div className="pt-3" style={{fontSize: '15px', letterSpacing:'5px',color:'white'}}>{this.props.subtitle}</div>
        </div>
      </div>
    );
  }
}

export default Header;
