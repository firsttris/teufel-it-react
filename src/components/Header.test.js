import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('Display Header', () => {
  const component = renderer.create(
    <Header title="[ teufel it ]" subtitle="Software ist unser Handwerk"></Header>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});