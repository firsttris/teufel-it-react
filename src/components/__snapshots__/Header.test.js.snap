// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Header 1`] = `
<div
  style={
    Object {
      "position": "absolute",
      "top": "200px",
      "width": "100%",
    }
  }
>
  <div
    className="p-3"
    style={
      Object {
        "backgroundColor": "#4B4B4B",
        "display": "inline-block",
      }
    }
  >
    <div
      style={
        Object {
          "color": "#f00",
          "fontSize": "35px",
          "letterSpacing": "8px",
        }
      }
    >
      [ teufel it ]
    </div>
    <div
      className="pt-3"
      style={
        Object {
          "color": "white",
          "fontSize": "15px",
          "letterSpacing": "5px",
        }
      }
    >
      Software ist unser Handwerk
    </div>
  </div>
</div>
`;
